### PR TITLE
[CI-SKIP] Add plugin developers section in README.md

### PR DESCRIPTION
**Pull request by CDFN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,44 @@ Download a copy of Waterfall.jar from our buildserver here: [Waterfall](https://
 
 Waterfall requires **Java 8** or above.
 
+## How To (Plugin Developers)
+------
+ * See our API patches [here](BungeeCord-Patches)
+ * Waterfall API javadocs here: [papermc.io/javadocs](https://papermc.io/javadocs/)
```
Comment (Chew) [inline README.md]: Should just lead to https://papermc.io/javadocs/waterfall, I understand Paper just leads to the javadocs page, but it has multiple options, and Waterfall just one.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,44 @@ Download a copy of Waterfall.jar from our buildserver here: [Waterfall](https://
 
 Waterfall requires **Java 8** or above.
 
+## How To (Plugin Developers)
+------
+ * See our API patches [here](BungeeCord-Patches)
+ * Waterfall API javadocs here: [papermc.io/javadocs/waterfall](https://papermc.io/javadocs/waterfall)
```
Comment (Proximyst) [inline README.md]:
```suggestion
 * Waterfall API JavaDocs here: <https://papermc.io/javadocs/waterfall>
```

File: README.md [hunk 3]
```diff
@@ -24,6 +24,44 @@ Download a copy of Waterfall.jar from our buildserver here: [Waterfall](https://
...
+ * Maven Repo (for waterfall-api):
```
Comment (Proximyst) [inline README.md]:
```suggestion
 * Maven repository (for `waterfall-api`):
```

File: README.md [hunk 4]
```diff
@@ -24,6 +24,44 @@ Download a copy of Waterfall.jar from our buildserver here: [Waterfall](https://
...
+```xml
+<repository>
+    <id>papermc</id>
+    <url>https://papermc.io/repo/repository/maven-public/</url>
+</repository>
+```
+ * Artifact Information:
```
Comment (Proximyst) [inline README.md]:
```suggestion
 * Artifact information:
```

**Review by Proximyst — APPROVED** [review]
Minor nitpicks.

**Review by electronicboy — APPROVED** [review]

**Pull request merged**